src/index.js

```diff
@@ -55,7 +55,7 @@ export function app(state, actions, view, container) {
   function resolveNode(node) {
     return typeof node === "function"
       ? resolveNode(node(globalState, wiredActions))
-      : node
+      : node || ''
```

Comment:
@jonlov This will fail when the node resolves to the number 0.

Let's try:

```js
node == null ? "" : node
```

Reply:
@jorgebucaran I change it. May I ask, why would ever a node be 0?

Reply:
@jonlov A node can be also a string or a number that will be casted as a TextNode. Maybe you have a view that returns the multiplication table and a specific subview that returns only a number. It's a shame we need to complicate the expression, but nodes can be strings or numbers, so if this code was written, e.g., in TypeScript we would get a type mismatch error.

Reply:
@jorgebucaran oh got it! Thanks for the explanation.